pysat/tests/test_utils.py

```diff
@@ -466,3 +468,47 @@ def test_netcdf_attribute_override(self):
         # custom attribute correctly read from file
         assert meta.bespoke
 
+
+class TestBasicNetCDF4xarray():
+    def setup(self):
```

Comment:
In theory this should be 
```
class TestBasicNetCDF4xarray(TestBasicNetCDF4):
```
to just repeat the above tests.  In reality this requires the `to_netcdf4` routine to be working with xarray, which is outside the scope of this PR.  Found a few bugs there and documented in #60.  

Using one basic test routine with a direct write from xarray rather than pysat to include some basic coverage for now.  We should fix this for 3.0.0.